src/index.js

```diff
@@ -7,11 +7,12 @@ const print = require("./printer");
 const languages = [
   {
     name: "Python",
+    since: "1.9.2", // FIXME: Fix this before releasing.
```

Comment:
Not actually correct but good enough for now. Need https://github.com/prettier/prettier/pull/3584 for the documented instructions to actually work.

Reply:
Not documented yet but this should be `prettier-python@0.0.0`

Reply:
it's for https://github.com/prettier/prettier/blob/3a9b25b8bc5f43401bacd1284eceeaa3295d6604/src/common/support.js#L16 – otherwise auto-inferring parser doesn't work since python gets filtered out